hedera-node/configuration/preprod/bootstrap.properties

```diff
@@ -2,4 +2,4 @@ bootstrap.throttleDefsJson.resource=throttles.json
 contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE,CONTRACT_ACTION
 hedera.recordStream.recordFileVersion=6
 hedera.recordStream.signatureFileVersion=6
-hedera.recordStream.enableTraceabilityMigration=true
\ No newline at end of file
+hedera.recordStream.enableTraceabilityMigration=false
```

Comment:
Is this related or should be changed in this release?

Reply:
yeah , should be changed back. Confirmed with @SimiHunjan